public_dropin_notebook_environments/python311_notebook_base/env_info.json

```
@@ -0,0 +1,11 @@
+{
+  "id": "664388ff6d426582042bb3e4",
+  "name": "[DataRobot] Python 3.11 Notebook Base image",
```

This might be a wildly small nit but maybe @nate-goudreault would appreciate - should we update the name of the image here: ??

```suggestion
  "name": "[DataRobot] Python 3.11 Notebook Base Image",
```

---

Thanks, Chris!

---

LGTM!